packages/components/src/styles/components/stepper/list.scss

```diff
@@ -0,0 +1,113 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+//
+// STEPPER > LIST
+//
+
+$hds-stepper-list-step-padding-bottom: 16px;
+
+.hds-stepper-list,
+.hds-stepper-list__step {
+  margin: 0;
+  padding: 0;
+  list-style: none;
+}
+
+// SUB COMPONENT: STEPPER > LIST > STEP
+
+.hds-stepper-list__step {
+  display: grid;
+  grid-column-gap: 8px;
+  grid-template-columns: auto 1fr;
+
+  &:not(:last-of-type) {
+    padding-bottom: $hds-stepper-list-step-padding-bottom;
+
+    .hds-stepper-list__step-progress {
+      &::before,
+      &::after {
+        position: absolute;
+        top: 1px;
+        left: 11px;
+        z-index: -1;
+        width: 2px;
+        content: "";
+      }
+
+      &::before {
```

Comment:
It would be helpful to label these pseudo element styles with comments explaining what they are used for